app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        self.add_person(self)
+
+    @classmethod
+    def add_person(cls, person: "Person"):
+        cls.people[person.name] = person
+
+    @classmethod
+    def get_person(cls, name: str) -> "Person":
+        return cls.people[name]
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    res = []
+    i = 0
+    for person_data in people:
+        person = Person(person_data["name"], person_data["age"])
+        res.append(person)
+    for person_data in people:
```

Comment:
If you want to have access to indices while iterating through an array, you can use [`enumerate()`](https://docs.python.org/3/library/functions.html#enumerate) function:
```python
for i, person_data in enumerate(people):
...
```